mgr/src/cmds/helpers.cr

```diff
@@ -105,3 +107,24 @@ def yes(label)
   value = STDIN.gets(chomp: true).not_nil!
   ["yes", "y", "yy", "ok", "sure", "on"].includes?(value.strip.downcase)
 end
+
+def handle_json_output(data, args)
+  if args.json
+    if data && data != ""
+      puts data.to_pretty_json
+      exit 0
+    else
+      puts "{}".to_json
+      exit 0
+    end
+  end
+end
+
+def handle_json_error(message, args)
```

Comment:
@aravindavk Sending err_msg in JSON. Have not tested this case.

Reply:
To test socket error. Stop the Moana server and then run the CLI